src/tenants/common/Home/index.tsx

```diff
@@ -41,7 +45,7 @@ export default function About({ leaderboard, tenantScore }: Props) {
           {t(`tenants:${config.tenantName}.title`)}
         </p>
 
-        {config.home.descriptionTitle && (
+        {config.home !== undefined && config.home.descriptionTitle && (
           <p
```

Comment:
There is still a warning here: 

`Property 'descriptionTitle' does not exist on type '{ image: string; }'.`